# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/DecoderFactory.cpp
# Commit: 1d5a44ae7207
# Full Hash: 1d5a44ae72071f079dcb00b62419c2ddd3c9f12c
# Author: Norisz Fay <nfay@mozilla.com>
# Date: 2023-01-27 21:11 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing build bustages on DecoderData.cpp CLOSED TREE
# ==============================================================================

diff -r 83609f9575ac -r 1d5a44ae7207 image/DecoderFactory.cpp
--- a/image/DecoderFactory.cpp	Fri Jan 27 19:06:16 2023 +0000
+++ b/image/DecoderFactory.cpp	Fri Jan 27 21:11:42 2023 +0200
@@ -217,7 +217,7 @@
   }
 
   MOZ_ASSERT(aType == DecoderType::GIF || aType == DecoderType::PNG ||
-                 aType == DecoderType::WEBP || aType == DecoderType::AVIF,
+                 aType == DecoderType::WEBP,
              "Calling CreateAnimationDecoder for non-animating DecoderType");
 
   // Create an anonymous decoder. Interaction with the SurfaceCache and the
@@ -272,7 +272,7 @@
   // rediscover it is animated).
   DecoderType type = aDecoder->GetType();
   MOZ_ASSERT(type == DecoderType::GIF || type == DecoderType::PNG ||
-                 type == DecoderType::WEBP || type == DecoderType::AVIF,
+                 type == DecoderType::WEBP,
              "Calling CloneAnimationDecoder for non-animating DecoderType");
 
   RefPtr<Decoder> decoder = GetDecoder(type, nullptr, /* aIsRedecode = */ true);